src/App.js

```diff
@@ -9,26 +9,43 @@ import { BrowserRouter as Router } from "react-router-dom";
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
-import {
-  Footer,
-  Navbar,
-} from "./components/molecules";
+import { Footer, Navbar } from "./components/molecules";
+import { DevelopmentPage } from "./components/pages";
+import { ScrollToTop } from "./components/atoms";
 // Routes
 import Routes from "./Routes";
 
+//> Configuration
+// Is the homepage ready to launch?
+const isLive = false;
+
 class App extends React.Component {
+  state = {
+    darkMode: false,
+  };
+
   render() {
-    return (
-      <Router>
-        <div className="flyout">
-          <Navbar />
-          <main>
-            <Routes />
-          </main>
-          <Footer />
-        </div>
-      </Router>
-    );
+    if (
+      window.location.hostname === "localhost" ||
+      window.location.hostname === "127.0.0.1" ||
+      isLive
```

Comment:
isLive could be changes to the real company domain.

Reply:
This should be a general switch and not be dependent on a certain domain name.